source/application/BioXAS/BioXASAppController.cpp

```diff
@@ -242,30 +219,18 @@ void BioXASAppController::goToEnergyCalibrationView(AMScan *toView)
 	}
 }
 
-void BioXASAppController::onCurrentScanActionStartedImplementation(AMScanAction *action)
-{
-	Q_UNUSED(action)
-
-	// Save current configuration to the database.
-
-	if (bioxasUserConfiguration_)
-		bioxasUserConfiguration_->storeToDb(AMDatabase::database("user"));
-}
-
 void BioXASAppController::onCurrentScanActionFinishedImplementation(AMScanAction *action)
 {
-	// Save current user configuration to the database.
-
-	if (bioxasUserConfiguration_)
-		bioxasUserConfiguration_->storeToDb(AMDatabase::database("user"));
-
 	// If the scan was an energy calibration scan, set the calibration view's scan and make it the current pane.
 
 	if (action && action->controller() && action->controller()->scan()) {
 		BioXASXASScanConfiguration *xasScanConfiguration = qobject_cast<BioXASXASScanConfiguration*>(action->controller()->scan()->scanConfiguration());
 		if (xasScanConfiguration && xasScanConfiguration->name() == "Energy Calibration XAS Scan")
 			goToEnergyCalibrationView(action->controller()->scan());
 	}
+
```

Comment:
I don't think it matters. The reason I did that is to adopt the ideas of constructor/destructor of O-O design"
```
Constructor order:
    super class
    sub calss

Destructor order:
    sub class
     super class
```
for this case, it is the actions we going to do when scan finished. So I put it in that order